docs/resources/pipeline_schedule.md

```diff
@@ -0,0 +1,35 @@
+# Resource: pipeline
```

Comment:
this should state `pipeline_schedule` not `pipeline` as this is what it actually it

Reply:
Fixed :)